docs/releasing/testing-and-linting.md

```diff
@@ -1,6 +1,6 @@
 # Testing and linting
 
-The CI lints SASS and JavaScript, and runs unit and functional tests with Node.
+The CI lints SASS and JavaScript, runs unit and functional tests with Node, and generates snapshots for visual regression testing (Percy.io).
```

Comment:
Will all our users know what 'visual regression testing' is? If not, do we need to link to an explanation?

Not sure about having 'Percy.io' in brackets at the end - there's no explanation of what it is (unless Percy.io = visual regression testing), and it isn't hyperlinked...maybe it would be ok to delete it from here, since we both hyperlink and explain it in line 53...